libcxx/include/chrono

```diff
@@ -998,7 +998,6 @@ constexpr chrono::year                                  operator ""y(unsigned lo
 #  include <cstdint>
 #  include <stdexcept>
 #  include <string_view>
-#  include <vector>
```

Comment:
Why did this have to be removed?

Reply:
It triggers a duplicate include warning with the one in the `_LIBCPP_STD_VER <= 20` block below.